perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/ModulesConfigLoader.java

```diff
@@ -0,0 +1,45 @@
+package cz.metacentrum.perun.core.impl.modules;
+
+import java.util.List;
+
+/**
+ * @author Vojtech Sassmann <vojtech.sassmann@gmail.com>
+ */
+public interface ModulesConfigLoader {
+
+	/**
+	 * For module with the given name, find a String configuration property with given name.
+	 *
+	 * @param moduleName name of a module
+	 * @param property name of a property
```

Comment:
You should mention that the property name can be chained by dots.

Reply:
Done.